src/tests/Interop/PInvoke/Int128/Int128Native.cpp

```diff
@@ -19,6 +19,12 @@
 
 static Int128 Int128Value = { };
 
+struct StructWithInt128
+{
+    int64_t messUpPadding;
```

Comment:
Maybe use `uint8_t` to better validate 32bit platforms that expect 4 byte alignment instead of 8/16 ones?

Reply:
it doesn't matter too much. The way alignment works in the various architectures we support/are likely to support, there won't be a meaningful difference. However, this is altogether messy enough that we may simply remove the ability to pass Int128 across the p/invoke boundary.